hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMRatisSnapshots.java

```diff
@@ -358,7 +367,8 @@ private void checkSnapshot(OzoneManager leaderOM, OzoneManager followerOM,
           Path leaderActiveSST =
               Paths.get(leaderActiveDir.toString(), fileName);
           // Skip if not hard link on the leader
-          if (!leaderActiveSST.toFile().exists()) {
+          // First confirm it is live
+          if (!liveSstFiles.stream().anyMatch(s -> s.equals(fileName))) {
```

Comment:
nit: can be simplified?

```suggestion
          if (!liveSstFiles.contains(fileName)) {
```